assets/wizards/popups/views/campaigns/index.js

```diff
@@ -0,0 +1,319 @@
+					<CustomSelectControl
+						label={ __( 'Campaigns', 'newspack' ) }
+						options={ [
+							{
+								key: 'active',
+								name: __( 'Active Prompts', 'newspack' ),
+								className: selectValue.key === 'active' && 'is-selected',
+							},
+							hasUnassigned && {
+								key: 'unassigned',
+								name: __( 'Unassigned Prompts', 'newspack' ),
+								className: selectValue.key === 'unassigned' && 'is-selected',
+							},
+							activeCampaigns.length && {
+								key: 'header-campaigns',
+								name: __( 'Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...activeCampaigns.map( ( { term_id: id, name } ) => ( {
+								key: id,
+								name,
+								className: classnames(
+									'newspack-campaigns__campaign-group__select-control-group-item',
+									+selectValue.key === +id && 'is-selected'
+								),
+							} ) ),
+							archivedCampaigns.length && {
+								key: 'header-archived-campaigns',
+								name: __( 'Archived Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...archivedCampaigns.map( ( { term_id: id, name } ) => ( {
+								key: id,
+								name,
+								className: classnames(
+									'newspack-campaigns__campaign-group__select-control-group-item',
+									+selectValue.key === +id && 'is-selected'
+								),
+							} ) ),
+						] }
+						onChange={ ( { selectedItem: { key } } ) =>
+							'active' === key
+								? history.push( '/campaigns' )
+								: history.push( `/campaigns/${ key }` )
+						}
+						value={ selectValue }
+						hideLabelFromVision={ true }
+					/>
+					{ campaignData && (
+						<div className="newspack-campaigns__campaign-group__filter-group-actions__button">
+							<Button
+								isQuaternary
+								isSmall
+								className={ popoverVisible && 'popover-active' }
+								onClick={ () => setPopoverVisible( ! popoverVisible ) }
+								icon={ moreVertical }
+								label={ __( 'Actions', 'newspack' ) }
+							/>
+							{ popoverVisible && (
+								<CampaignManagementPopover
+									dismiss={ () => setPopoverVisible( false ) }
+									isArchive={ campaignData && 'archive' === campaignData.status }
+									onActivate={ () => manageCampaignGroup( prompts ) }
+									onArchive={ () => archiveCampaignGroup( campaignId, true ) }
+									onDeactivate={ () => manageCampaignGroup( prompts, 'DELETE' ) }
+									onDelete={ () => deleteCampaignGroup( campaignId ) }
+									onDuplicate={ () => {
+										setModalVisible( true );
+										setCampaignName( '' );
+										setModalType( MODAL_TYPE_DUPLICATE );
+									} }
+									onRename={ () => {
+										setCampaignName( campaignData.name );
+										setModalVisible( true );
+										setModalType( MODAL_TYPE_RENAME );
+									} }
+									onUnarchive={ () => archiveCampaignGroup( campaignId, false ) }
+									hasUnpublished={ prompts.some( ( { status } ) => 'publish' !== status ) }
```

Comment:
You're right, but because this gets a bit more complex to resolve (the UI fix would just be a bandaid), why don't we wait for https://github.com/Automattic/newspack-plugin/pull/832